<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>699</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zodiac">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Zodiac</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>90</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X_Center</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_xCenter">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y_Center</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_yCenter">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_submitViewChange">
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_replay">
      <property name="text">
       <string>REPLAY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_adjust">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Zoom In: Mouse Up</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zoom Out: Mouse Down</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Accelerate: A + Mouse Up</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decelerate: A + Mouse Down</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Set Center : L + Mouse Click</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Display Center: L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>See Planet Info: Mouse Clcik</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>READ ME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X_OFFSET</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_xoffset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y_OFFSET</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_yoffset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LOCK ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_pLocked">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_undoChange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>UNDO</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_history">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HISTORY</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
